VideoLocker/src/main/java/org/edx/mobile/module/registration/view/RegistrationCheckBoxView.java

```diff
@@ -82,24 +74,27 @@ public View getView() {
     }
 
     @Override
-    public void handleError(String error) {
+    public void handleError(String error, boolean requestAccessibility) {
```

Comment:
I suppose, I could make a function that passes false as a default. But nothing would call that function. thoughts?